README.md

```diff
@@ -10,8 +10,14 @@ for both the composer and capi AWS accounts from [janus](https://janus.gutools.c
 
  - Fetch config from S3: `./fetch-config.sh`
  - If you get an error message saying that you requred AWS Signature Version 4, configure your aws cli by running `aws configure set default.s3.signature_version s3v4`
- - Setup the nginx mapping by following the instructions in the
+ - Setup the nginx mapping by following the instructions in the "Install config for an application" section of the 
  [dev-nginx readme](https://github.com/guardian/dev-nginx#install-config-for-an-application).
+ 
+    tl;dr? Try this...
+    * Clone the [dev-nginx](https://github.com/guardian/dev-nginx) private repo to your machine (e.g. ~/code/dev-nginx)
+    * Change directory to that location: `cd ~/code/dev-nginx`
+    * Assuming your copy of atom-workshop is in ~/code/atom-workshop, run `sudo ./setup-app.rb ~/code/atom-workshop/nginx/nginx-mapping.yml`
```

Comment:
shouldn't need `sudo` here anymore

Reply:
Is that recent? I had to `sudo` that a few days ago or it wouldn't run - had permission denied errors.

Reply:
It's at least a couple of months old - I did it as part of the move to `mkcert` in dev-nginx.

`setup-app.rb` will write a file to `/usr/local/etc/nginx/servers` and then restart nginx. Your user has permission to write to that directory and the [restarting of nginx](https://github.com/guardian/dev-nginx/blob/master/restart-nginx.sh) invokes `sudo` and will there prompt for a password when needed.

If you're having to run `sudo ./setup-app.rb` it would suggest that something has previously been installed as `sudo` and thus has the wrong permissions. That could escalate...